slc/sample-app/thermostat/efr32/thermostat-thread.slcp

```diff
@@ -85,39 +85,55 @@ component:
     from: matter
   - id: matter_thread_network_diagnostics
     from: matter
-  - id: matter_time_format_localization
-    from: matter
   - id: matter_user_label
     from: matter
   - id: matter_unit_localization
     from: matter
-  - id: matter_wifi_network_diagnostics
-    from: matter
   - id: matter_default_lcd_config
     from: matter
   - id: matter_log_uart
     from: matter
   - id: matter_si70xx_sensor
     from: matter
     condition: [hardware_board_has_si70xx]
+  - id: matter_zigbee_multiprotocol_common
```

Comment:
`matter_zigbee_multiprotocol_common` could require `matter_zigbee_cmp` or vice versa. Or we regroup it in one file